proposals/0274-add-preferred-FPS.md

```diff
@@ -32,7 +32,7 @@ Add `preferredFPS` to `VideoStreamingCapability` struct in both APIs.
         ...
         <!-- new param -->
         <param name="preferredFPS" type="Integer" minvalue="0" maxvalue="2147483647" mandatory="false">
-            <description>Preferred frame rate per second. Mobile application should take this value into account for capturing and encoding video frame.</description>
+            <description>Preferred frame rate per second. Mobile application should take this value into account for capturing and encoding video frame, but mobile application should also take care of the case where mobile device's hardware performance is constrained. It is up to mobile application to determine the actual frame rate.</description>

```

Comment:
```suggestion
            <description>Preferred frame rate per second. Mobile application should take this value into account for capturing and encoding video frame, but the app library should also take care of the case where the mobile device's hardware performance is constrained. It is up to the app library to determine the actual frame rate.</description>

```

Reply:
Well, the suggested change does not correct. If iOS SDL application that uses SDLCarWindow and LifecycleManager, sdl_ios library will take care of the frame rate. In other words, frame rate is still up to mobile application if some iOS apps that do not use SDLCarWindow and/or Android application running with current sdl_java_suite library. I have no good idea how to explain the situation appropriately, however.

Reply:
What about this instead?

`The preferred frame rate per second of the head unit. The mobile application / app library may take other factors into account that constrain the frame rate lower than this value, but it should not streaming at a higher FPS than this value.`

Reply:
@joeljfischer, thanks for the suggestion. Sounds good to me.